lgn-messages/src/routing/mod.rs

```diff
@@ -1,27 +0,0 @@
-//! Message routing high level semantic.
-use serde_derive::Deserialize;
-use serde_derive::Serialize;
-
-#[derive(Clone, Debug, PartialEq, Eq, Hash, Deserialize, Serialize)]
-pub struct RoutingKey {
-    domain: String,
-    priority: u64,
-}
```

Comment:
Routing is no longer used by the GW, now the GW has its protobuf based messages and it uses the task class.